testing/stereo-preview/app.py

```diff
@@ -0,0 +1,78 @@
+import logging
+import depthai as dai
+import cv2
+import numpy as np
+
+
+class DepthVisualizer:
+    WLSLevel = {"LOW": (1000, 0.8), "MEDIUM": (6000, 1.5), "HIGH": (12000, 2.0)}
+
+    def __init__(self, max_disparity: int):
+        self.max_disparity = max_disparity
+        self._config_wls()
+
+    def _config_wls(self, wls_level: str = "HIGH", wls_lambda: float = None, wls_sigma: float = None) -> None:
+        default_lambda, default_sigma = self.WLSLevel.get(wls_level.upper(), self.WLSLevel["HIGH"])
+        self.wls_config = {
+            "enabled": True,
+            "lambda": wls_lambda or default_lambda,
+            "sigma": wls_sigma or default_sigma,
+        }
+
+    def _apply_wls_filter(self, left_disp: np.ndarray):
+        # TODO: Implement WLS filter for disparity map with visualization - ask @PetrNovota or @Zimamazim
+        pass
```

Comment:
I was looking to apply the WLS filter to disparity maps but couldn't find any specifics in docs.  I came across the config_wls method in our [SDK docs here](https://docs.luxonis.com/projects/sdk/en/latest/components/stereo_component/?highlight=WLS#depthai_sdk.components.StereoComponent.config_wls) , but I'm still unclear on where exactly the WLS filter should be applied or it is applied within StereoDepth in post-processing step. ![StereoDepth in post-processing step](https://docs.luxonis.com/projects/api/en/latest/_images/depth_diagram.png)

Could you please point me in the right direction? @PetrNovota @Zimamazim